x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/actions_log_date_range_picker.tsx

```diff
@@ -69,6 +69,20 @@ export const ActionLogDateRangePicker = memo(
       );
     });
 
+    const endDate = useMemo(
+      () =>
+        isFlyout ? dateRangePickerState.endDate : endDateFromUrl ?? dateRangePickerState.endDate,
+      [dateRangePickerState.endDate, endDateFromUrl, isFlyout]
+    );
+
+    const startDate = useMemo(
+      () =>
+        isFlyout
+          ? dateRangePickerState.startDate
+          : startDateFromUrl ?? dateRangePickerState.startDate,
+      [dateRangePickerState.startDate, isFlyout, startDateFromUrl]
+    );
+
```

Comment:
can this also be removed maybe? simply passing `dateRangePickerState.(start|end)Date` to `EuiSuperDatePicker`?

Reply:
I don't see why not 🎉